apps/frontend/package.json

```
@@ -3,11 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@sovryn/ethers-provider": "*",
+    "@sovryn/tailwindcss-config": "*",
+    "@sovryn/ui": "*",
+    "ethers": "^5.7.1",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
```

```suggestion
    "ethers": "5.7.1",
    "react": "18.1.0",
    "react-dom": "18.1.0",
```

See https://sovryn.monday.com/docs/2733202752?blockId=aebe8f4b-5e4a-4ef6-8967-503ce7a1bb30 for explanation.